src/issued/contractInterface.ts

```diff
@@ -1,31 +1,22 @@
 import { constants } from "ethers";
-import { Hash, OpenAttestationContract } from "../types";
-import { TYPES } from "../common/smartContract/constants";
+import { v3 } from "@govtechsg/open-attestation";
```

Comment:
Shall we remove this from the `issued` folder and put them with the verifiers?

Reply:
moved under verifiers/documentStoreIssued